client/src/components/Login/Login.js

```diff
@@ -39,22 +42,18 @@ const styles = {
 const useStyles = makeStyles(styles)
 
 const Login = props => {
+  const userForm = useRef(null)
+
   const { addToast } = useToasts()
-  const [username, setUserName] = useState('')
-  const [password, setPassword] = useState('')
   const [redirect, setRedirect] = useState(false)
   const dispatch = useDispatch()
   const userInfo = useSelector(state => state.loginReducer.currentUser)
```

Comment:
Can you please use selector functions here?

Reply:
Done